apply/run.go

```diff
@@ -14,6 +14,7 @@ import (
 
 type ClusterArgs struct {
 	cluster    *v1.Cluster
+	imageName  string
```

Comment:
this args should be from cmd/run.go transfer to here. I not found `imageName` args in cmd/run command, you not finished yet?